tests/expectations.json

```diff
@@ -457,22 +457,12 @@
   "built-ins/Array/prototype/toSpliced/unmodified.js": "CRASH",
   "built-ins/Array/prototype/toString/non-callable-join-string-tag.js": "CRASH",
   "built-ins/Array/prototype/unshift/S15.4.4.13_A1_T1.js": "CRASH",
-  "built-ins/Array/prototype/unshift/S15.4.4.13_A1_T2.js": "CRASH",
   "built-ins/Array/prototype/unshift/S15.4.4.13_A2_T1.js": "CRASH",
   "built-ins/Array/prototype/unshift/S15.4.4.13_A2_T2.js": "CRASH",
-  "built-ins/Array/prototype/unshift/S15.4.4.13_A2_T3.js": "CRASH",
   "built-ins/Array/prototype/unshift/S15.4.4.13_A3_T2.js": "CRASH",
```

Comment:
**issue (nonblocking):** It seems we're not treating negative lengths in the way that the spec expects. Not sure where the issue lies though. This is the code it's running:

```js
var obj = {};
obj.unshift = Array.prototype.unshift;
obj[0] = "";
obj.length = -4294967295;

var unshift = obj.unshift("x", "y", "z");
```

Could you check what the crash is coming from using `eval-test` and/or a small test file like a `test.js`? And if the issue is in the unshift implementation then of course fix it. If it is not, then possibly make an issue about it.